### PR TITLE
test(cli): add tests for core/kubectl module

### LINKED ISSUE #549: CLI[Test]: Improve test coverage for core/kubectl module

### PR DESCRIPTION
**Pull request by qnswirlds**
## Description

This pull request changes the following:

- Add tests for core/kubectl module

### Related Issues

- Closes #549

### CODE REVIEW

File: fullstack-network-manager/src/core/kubectl.mjs [hunk 1]
```diff
@@ -137,8 +137,8 @@ export class Kubectl extends ShellRunner {
    * @param bashScript bash script to be run within the container (e.g 'ls -la /opt/hgcapp')
    * @returns {Promise<Array>} console output as an array of strings
    */
-  async execContainer (pod, container, bashScript) {
-    return this.exec(pod, `-c ${container} -- `, `bash -c "${bashScript}"`)
+  async execContainer(pod, container, bashScript) {
+    return this.exec(pod, `-c ${container}`, `-- bash -c "${bashScript}"`)
```
Comment (qnswirlds) [inline fullstack-network-manager/src/core/kubectl.mjs:141]: I think it's clearer this way.

File: fullstack-network-manager/src/core/kubectl.mjs [hunk 2]
```diff
@@ -147,7 +147,7 @@ export class Kubectl extends ShellRunner {
    * @param args args of the command
    * @returns {Promise<Array>} console output as an array of strings
    */
-  async copy (pod, ...args) {
+  async copy(...args) {
```
Comment (qnswirlds) [inline fullstack-network-manager/src/core/kubectl.mjs:150]: Removed unused `pod`

Comment (leninmehedy) [reply fullstack-network-manager/src/core/kubectl.mjs:150]: We need to force the first arg to be the pod name

File: fullstack-network-manager/src/core/kubectl.mjs (hunk 1 above)
Comment (leninmehedy) [reply fullstack-network-manager/src/core/kubectl.mjs:141]: ok

**Review by leninmehedy — COMMENTED** [review]
could you please run `npm run format` to fix the lint errors.

File: fullstack-network-manager/src/core/platform_installer.mjs [hunk 3]
```diff
@@ -78,7 +78,7 @@ export class PlatformInstaller {
     if (!fs.statSync(buildZipFile).isFile()) throw new IllegalArgumentError('buildZipFile does not exists', buildZipFile)
 
     try {
-      await this.kubectl.copy(podName,
+      await this.kubectl.copy(
```
Comment (leninmehedy) [inline fullstack-network-manager/src/core/platform_installer.mjs:81]: This is incorrect as podName is required

File: fullstack-network-manager/src/core/platform_installer.mjs [hunk 4]
```diff
@@ -99,7 +99,7 @@ export class PlatformInstaller {
     try {
       for (const srcPath of srcFiles) {
         self.logger.debug(`Copying files into ${podName}: ${srcPath} -> ${destDir}`)
-        await this.kubectl.copy(podName,
+        await this.kubectl.copy(
```
Comment (leninmehedy) [inline fullstack-network-manager/src/core/platform_installer.mjs:102]: same here, podName is required

**Review by leninmehedy — APPROVED** [review]
Please create a separate ticket to fix the function definition `kubectl.copy` to accept two clear arguments as: `srcPath` and `dstPath`.

**Pull request merged**